articles/tokens/id-token.md

```diff
@@ -0,0 +1,88 @@
+---
+title: ID token
+description: How to obtain, use and renew an id_token.
+---
+
+# ID Token
+
+## Overview
+
+The ID token, usually referred to as `id_token` in code samples, is a [JSON Web Token (JWT)](/jwt) that contains user profile attributes represented in the form of _claims_. These claims are statements about the user, which can be trusted if the consumer of the token can verify its signature (which was generated with the Auth0 client's _Client Secret_). The `id_token` is consumed by the client and used to get user information like the user's name, email, and so forth, typically used for UI display. It was added to the OIDC specification as an optimization so the client can know the identity of the user, without having to make an additional network requests.
```

Comment:
The ID token can also be signed with a private key and verified with a public key - some clients such as SPAs are not capable of storing secrets so they don't have a client secret at all.